colorbleed/plugins/maya/load/load_yeti_cache.py

```diff
@@ -124,14 +124,19 @@ def update(self, container, representation):
 
         path = api.get_representation_path(representation)
         members = cmds.sets(container['objectName'], query=True)
-        yeti_node = cmds.ls(members, type="pgYetiMaya", long=True)
+        all_members = cmds.listRelatives(members, ad=True)
```

Comment:
Actually, wait.. no! This means the problem *must!* be elsewhere - the container should always contain all content!

Reply:
> Actually, wait.. no! This means the problem must! be elsewhere - the container should always contain all content!

When a Yeti node''s transform node, similar to a mesh's transfrom node, is added we don't get the `pgYetiMaya` or `pgYetiGroom` nodes. In hindsight you are correct, this will mean that the loading of representation into the scene is incorrect. I will improve this asap!